pkg/scheduler/actions/factory.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package actions
 
 import (
+	"volcano.sh/volcano/pkg/scheduler/actions/elect"
+	"volcano.sh/volcano/pkg/scheduler/actions/reserve"
```

Comment:
it's better to put these packages together with the following plugin packages :)

Reply:
done